External/FEXCore/Source/Interface/Core/JIT/x86_64/JIT.cpp

```diff
@@ -1781,32 +1784,31 @@ void *JITCore::CompileCode([[maybe_unused]] FEXCore::IR::IRListView<true> const
           // Result: RAX
 
           // These are pushed in reverse order because stacks
-          for (uint32_t i = 7; i > 0; --i)
+          for (uint32_t i = 7; i > 0; --i) {
```

Comment:
`FEXCore::HLE::SyscallArguments::MAX_ARGS`?

Reply:
Did it